pkg/console/console.go

```diff
@@ -92,7 +96,47 @@ func Run() error {
 		},
 	}
 
-	return server.ListenAndServeTLS("", "")
+	sigCtx, cancel := signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
+	defer cancel()
+	group, groupCtx := errgroup.WithContext(sigCtx)
+
+	group.Go(startWatch(groupCtx, watch))
+	group.Go(startServer(server))
+	group.Go(stopServer(groupCtx, server))
```

Comment:
```suggestion
	group.Go(startWatch(watch, groupCtx))
	group.Go(startServer(server))
	group.Go(stopServer(server, groupCtx))
```

Although unusual, it might be nicer to read this way.